plugins/by-name/lightline/default.nix

```diff
@@ -22,20 +22,20 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
        settings.component_function = {
          readonly = "LightlineReadonly";
        };
-     };
 
-     extraConfigLua = '''
-       function LightlineReadonly()
-         local is_readonly = vim.bo.readonly == 1
-         local filetype = vim.bo.filetype
+       luaConfig.pre= '''
```

Comment:
I'm sure the tests should have picked up if this was wrong, but shouldn't this be nested under the plugin's namespace?

E.g. `plugins.ligntline.luaConfig.pre`?

(Same for other two)

Reply:
Oh, sorry. I missed the indentation change and the removed `}` above.